CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 3.2.0 (2022-xx-xx)
 
+* Increased minimum PHP version to 8.0 (to be consistent with KnpMenu 3.5)
```

Comment:
We should not have 2 `Increased minimum PHP version to` in the changelog for the same version of the bundle.

Reply:
+1 for dropping Symfony 4.4

Reply:
3.2.0 has already been released, we forgot to update the changelog.
Please fix it, and the new entry as 3.3.0

Reply:
I changed it with the help of https://github.com/KnpLabs/KnpMenuBundle/releases . But as there is no KnpMenu release 3.4 for php 7.4 (as planned)
https://github.com/KnpLabs/KnpMenu/releases

Please have a look at the new version.